### ISSUE TITLE
Real Names

### ISSUE THREAD

**Issue by XBessa** [NuSkooler/enigma-bbs#101]
To be able to have real names into certain echo mail areas.

**Comment by NuSkooler** [comment]
@XBessa You may have answered this on HappyNet (I can check in a bit) but is this causing immediate issue? If so I'll take a look at it sooner.

**Comment by XBessa** [comment]
Nope, you just asked me in Happynet to raise an issue about it at GitHub,
but no worries, there's no rush into that.

Em sábado, 1 de outubro de 2016, Bryan Ashby <notifications@github.com>
escreveu:

**Comment by NuSkooler** [comment]
Resolved @ 0a98ce651ff40c29507b831564b27a5afe627f47

To configure an area for real names, set `realNames` to true, e.g.:
```
areas: {
  // ...
  some_area_tag: {
    name: Some Area
    desc: This is a area with Real Names
    realNames: true
  }
}
```
